pathogen-characterisation/quality-control.md

```diff
@@ -18,14 +16,125 @@ training:
+    * sample starting volume; 
+    * pathogen concentration methods / protocols if any;
+    * extraction methods (protein, DNA, RNA, etc), if any. Keep the information about the devices/technology/kits used, starting volume of sample concentrate, elution volume following extraction, etc;
+    * hardware/instruments used.
+
+* From the analytical methods:
+    * target sequence, if any;
+    * analytical methods (ie PCR, qPCR, etc);
+    * information about the enzymes/reagents/kits used (ie. concentrations, volumes, etc.);
+    * analytical steps and/or conditions (ie. temperatures, time, number of PCR cycles, etc);
+    * hardware/instruments used;
+
+### Existing approaches
+
+For the analysis part, especially if it is done in different facilities or even countries, reference standards should be used when available, choosing the ones specific for the domain of study when possible.
+
+Quality assurance / quality control (QA/QC) measures should be always in place when performing the analyses, and they are especially relevant when no standard operating procedures are available or followed.
+
+* Some quality measures that may be:
+    * Consider the use of any reference material, for instance, samples having quantified pathogen/viral material to compare with the samples that are being analysed and/or procedures that are  used.
+    * Always include a negative and a positive control sample and document the ones that are being used.
+    * Consider to follow standardised protocols when possible and also reference it in the metadata.
+
+Using wastewater samples for pathogen surveillance might benefit from the following quality control steps:
+* Quality control of the preanalytical process
+    * Spiking wastewater samples with exogenous calibrated virus reference material, such as mengovirus or murine norovirus, before preanalytical processing in order to evaluate the efficiency of viral concentration and extraction procedures (by evaluating the recovery of the exogenous reference material introduced into the sample prior to preanalytical processing). 
+    * Including a negative control sample during the nucleic acid extraction procedure to evaluate for potential cross contamination.
+* Quality control of the quantitative analytical detection
+    * Standard calibrated reference material (such as EURM-019 / EURM-014 single stranded RNA provided by the Joint Research Center)
+
+In particular the European Commission Recommendation (EU) 2021/472 of 17 March 2021 has advocated the use of a common approach to establish a systematic surveillance of SARS-CoV-2 and its variants in wastewaters in the EU (OJ L 98 19.03.2021, p. 3, CELEX: Specific Quality Standards are indicated in this Recommendation document: standards for PCR/Digital-PCR (polymerase chain reaction), shortly described below:
```

Comment:
>EU (OJ L 98 19.03.2021, p. 3, CELEX: Specific Quality Standards are indicated in this Recommendation document: standards for PCR/Digital-PCR (polymerase chain reaction), shortly described below:

This sentence is missing a parenthesis and is difficult to read.

Reply:
Indeed, thanks! I've modified it